client/src/app/pages/user-settings/user-settings.component.html

```diff
@@ -0,0 +1,63 @@
+<div class="max-w-3xl mx-auto px-4 mt-8">
+  <p-card class="border border-surface-200 rounded-lg">
```

Comment:
<p-card class="border border-surface-200 rounded-lg">
```suggestion
  <p-card class="rounded-lg">
```